app/src/main/java/com/tohami/newsapi/base/BaseFragment.java

```diff
@@ -0,0 +1,32 @@
+package com.tohami.newsapi.base;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+public abstract class BaseFragment extends Fragment {
+    protected abstract View initializeViews(int layoutId , LayoutInflater inflater,  ViewGroup container,  Bundle savedInstanceState);
+
+    protected abstract void setListeners();
+    protected abstract int getLayoutId() ;
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
```

Comment:
Remove this, it does not perform any functionality

Reply:
done